src/main/res/values/strings.xml

```diff
@@ -23,14 +23,14 @@
     <string name="compiled_for_debugging">Compiled for debugging</string>
     <string name="confirmation">Confirmation</string>
     <string name="connecting">Connecting…</string>
-    <string name="contains_non_free_media">Contains non-free media</string>
+    <string name="contains_non_free_media">Contains non-libre media</string>
     <string name="could_not_download_FORMAT">Could not download %s</string>
     <string name="could_not_sync_FORMAT">Could not sync %s</string>
     <string name="could_not_validate_FORMAT">Could not validate %s</string>
-    <string name="credits">Credits</string>
+    <string name="credits">Contributors</string>
     <string name="dark">Dark</string>
-    <string name="delete">Delete</string>
-    <string name="delete_repository_DESC">Delete the repository?</string>
```

Comment:
The action is actually removing the repo and not disabling it, so I guess neither is right, "Remove repository?" is possibly the better solution

Reply:
It only being removed on the user end is only obvious if you already know that.
There is nothing leading the user to understand what a repository is.

Reply:
Yeah, this is why disabling isn't precise, it'll get removed from the list fully not just only disabled.

Reply:
The same applies to this as the above one (should be kept delete)